src/components/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -62,6 +45,31 @@ const StyledBreadcrumbs = styled.nav`
   }
 `;
 
+const StyledLink = styled.a<{ isDisabled?: boolean }>`
+  line-height: 1.125rem;
+  font-size: ${theme.fontSizes.sm};
+  font-weight: ${theme.fontWeights.regular};
+  font-family: ${theme.fonts.body};
+  text-decoration: none;
+  transition: ${theme.transition};
+  color: ${(props) =>
+    props.isDisabled ? theme.color.text.text02 : theme.color.interactive.link};
+  pointer-events: ${(props) => (props.isDisabled ? "none" : "auto")};
```

Comment:
The styles for `disabled` state should be defined in a separate variable and conditionally applied all at once.
Mixing every style here and adding conditional statements for every line increases complexity and reduces readability.